builder/proxmox/config_test.go

```diff
@@ -8,6 +8,17 @@ import (
 	"github.com/hashicorp/packer/template"
 )
 
+func mandatoryConfig(t *testing.T) map[string]interface{} {
```

Comment:
Added this small helper function to provide mandatory builder configuration. Test methods become a wee bit shorter and a bit clearer as to what is being asserted 🕵️